### PR TITLE
Long/cd

### PR DESCRIPTION
**Pull request by LongOddCode**

### CODE REVIEW

**Comment by LongOddCode** [comment]
The schedule is UTC time. So our daily release will be triggered at 16:00 Beijing time.

**Review by henzhang-ms — APPROVED** [review]
approve for modification of templates prerelease

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -101,18 +107,13 @@ jobs:
...
 
-      - name: release daily npm packages to npmjs.org
-        if: ${{ github.ref == 'refs/heads/dev' && startsWith(github.event.inputs.version, 'pre') }}
-        run: |
-          npx lerna publish --no-private --preid=alpha --dist-tag=alpha ${{ github.event.inputs.version }} --yes ${{ github.event.inputs.force }}
-
-      - name: release rc npm packages to npmjs.org
-        if: ${{ startsWith(github.ref, 'refs/heads/release/') && github.event.inputs.version == 'prerelease' }}
+      - name: release prerelease npm packages to npmjs.org
+        if: ${{ startsWith(github.event.inputs.version, 'pre') }}
```
Comment (qinezh) [inline .github/workflows/cd.yml:111]: Does it mean daily release and pre release can be published from arbitrary branches? Do we need to restrict the branch like the stable release?

Comment (LongOddCode) [reply .github/workflows/cd.yml]: I’ve already added some validations. Thanks bro.

**Review by qinezh — APPROVED** [review]

**Pull request merged**